recipe/meta.yaml

```diff
@@ -1,19 +1,19 @@
 {% set name = "openmm-atmmetaforce-plugin" %}
-{% set version = "0.3.4" %}
+{% set version = "0.3.5" %}
 
 package:
   name: {{ name }}
   version: {{ version }}
 
 source:
   url: https://github.com/Gallicchio-Lab/{{ name }}/archive/refs/tags/v{{ version }}.tar.gz
-  sha256: fde821107efd4b47eed26c9eb9d07bff78ca219b3c13c796bc60253b3e61c3a1
+  sha256: a6e96971d9d235a821653513123d3bd9876ad2bf540700e42327f668e96502ab
   patches:
     # py >=3.9 builds fail on OSX if pip isn't used to install the python package
     - patches/patch-install-no-deps-osx.patch  # [osx]
```

Comment:
```suggestion
```

If you delete these lines and delete the corresponding file then that should hopefully fix things!